lib/CL/devices/cuda/pocl-ptx-gen.cc

```diff
@@ -1053,3 +1056,26 @@ int pocl_cuda_get_ptr_arg_alignment(void *LLVM_IR, const char *KernelName,
   std::memcpy(Alignments, AVec.data(), sizeof(size_t) * AVec.size());
   return 0;
 }
+
+int pocl_cuda_override_sub_group_size(void *llvm_module, int SGSize) {
```

Comment:
A bit misnomer here as the _pocl_sub_group_size is not defined IUCC. So maybe:
```suggestion
int pocl_cuda_define_sub_group_size(void *llvm_module, int SGSize) {
```